app/commands/user_track/generate_concept_status_mapping.rb

```diff
@@ -5,6 +5,8 @@ class GenerateConceptStatusMapping
     initialize_with :user_track
 
     def call
+      return {} if user_track.nil?
```

Comment:
Thanks :)

Reply:
This is more idiomatic:

```suggestion
      return {} unless user_track
```

Reply:
done